torchci/rockset/pytorch_dev_infra_kpis/__sql/ci_wait_time.sql

```diff
@@ -0,0 +1,74 @@
+WITH percentiles_desired AS (
+  -- All the percentiles that we want the query to return
+  SELECT 'p25' as percentile, 0.25 as percentile_num
+  UNION ALL
+  SELECT 'p50', 0.50
+  UNION ALL
+  SELECT 'p75', 0.75
+  UNION ALL
+  SELECT 'p90', 0.90
+),
+-- Set the bucket to the desired granularity
+granular_pr_stats as (
+    SELECT
+      DATE_TRUNC(:granularity, end_time) AS bucket,
+      *
+    FROM metrics.pr_stats
+),
+-- Within each bucket, figure out what percentile duration and num_commits each PR falls under
+percentiles as (
+  SELECT
+      pr_number,
+      bucket,
+      duration_mins,
+      PERCENT_RANK() OVER(
+          PARTITION BY bucket
+          ORDER by duration_mins
+      ) as duration_percentile,
+      num_commits,
+      PERCENT_RANK() OVER(
+          PARTITION BY bucket
+          ORDER by num_commits
+      ) as num_commits_percentile
+  FROM
+      granular_pr_stats
+  WHERE 1 = 1
```

Comment:
This is a formatting thing, which lets two things happen:
1. All the following conditions are indented consistently
2. During protyping, commenting out the first condition for testing/etc is trivial

Reply:
This is a readability thing, so that the actual conditions are all indented the same way. It also has the side benefit when editing the query of making it really easy to comment out the first condition in the statement (no need to remove the `AND` on the next line)